src/test/java/domain/AnswerLottoTest.java

```diff
@@ -16,6 +17,7 @@ void test() {
 
         // when & then
         Assertions.assertThatIllegalArgumentException().isThrownBy(() -> new AnswerLotto(lotto, 6));
-        Assertions.assertThatIllegalArgumentException().isThrownBy(() -> new AnswerLotto(lotto, 46));
+        Assertions.assertThatIllegalArgumentException()
+                .isThrownBy(() -> new AnswerLotto(lotto, LottoNumber.MAX_RANGE.getNumber() + 1));
```

Comment:
일반적으로 assertThatThrownBy() 를 더 많이 사용합니다 😄 
```suggestion
        Assertions.assertThatThrownBy(() -> new AnswerLotto(lotto, LottoNumber.MAX_RANGE.getNumber() + 1))
                        .isInstanceOf(IllegalArgumentException.class)
                                .hasMessageContaining("1~45 범위 이내여야 합니다.");
```